qtdialogs/DlgWelcomeScreen.py

```diff
@@ -0,0 +1,359 @@
+        btnLayout = QtWidgets.QHBoxLayout(btnContainer)
+        btnLayout.setContentsMargins(0, 0, 0, 0)
+
+        # Get Started Button
+        self.getStartedBtn = createStyledButton('Get Started', UI_STYLE_BUTTON_STANDARD)
+        self.getStartedBtn.clicked.connect(self.accept)
+
+        btnLayout.addStretch()
+        btnLayout.addWidget(self.getStartedBtn)
+        btnLayout.addStretch()
+
+        return btnContainer
+
+    def createVersionSection(self):
+        """Create the version and copyright information section."""
+        versionText = createStyledLabel("""
+            <div style="color: %s; font-size: 8pt; text-align: center; line-height: 1.2;">
+            Armory Bitcoin Wallet: Version 0.96.00-beta<br>
+            © 2025 Goatpig
+            </div>
+        """ % htmlColor('Foreground'), QtCore.Qt.AlignCenter)
+        return versionText
+
+    def updateSystemRequirements(self, requirements):
+        """Update the system requirements display with provided values."""
+        def createStatusText(requirement, name):
+            """Create styled status text for a requirement."""
+            color = htmlColor('TextGreen') if requirement else htmlColor('TextRed')
+            status = 'Sufficient' if requirement else 'Insufficient'
+            return f'<div style="line-height: 1.2;">' \
+                   f'<span style="color: {color}; font-size: 10pt;">✓ {name}: {status}</span>' \
+                   f'</div>'
+
+        # Update system requirement displays
+        self.diskLabel.setText(createStatusText(
+            requirements['disk_space'], 'Storage Space'))
+        self.ramLabel.setText(createStatusText(
+            requirements['ram'], 'System Memory'))
+        self.cpuLabel.setText(createStatusText(
+            requirements['cpu_cores'], 'CPU Cores'))
+
+    def checkSystemRequirements(self):
+        """Check if the system meets minimum requirements for running Armory."""
+        requirements = {
+            'disk_space': False,
+            'ram': False,
+            'cpu_cores': False
+        }
+
+        try:
+            requirements['disk_space'] = self._checkDiskSpace()
+            requirements['ram'] = self._checkSystemMemory()
+            requirements['cpu_cores'] = self._checkCpuCores()
+        except Exception as e:
+            print(f"Error checking system requirements: {str(e)}")
+            # If we can't check requirements, assume they're sufficient
+            return self._getDefaultRequirements()
+
+        return requirements
+
+    def _checkDiskSpace(self):
+        """Check if sufficient disk space is available."""
+        try:
+            if os.name == 'nt':  # Windows
+                return self._checkDiskSpaceWindows()
+            else:  # Unix-like systems
+                return self._checkDiskSpaceUnix()
+        except Exception:
+            return True  # Assume sufficient if check fails
+
+    def _checkDiskSpaceWindows(self):
```

Comment:
the system checks should be implement in armoryengiune/ArmoryUtils.py, if they're not there already